WalletWasabi.Fluent/ViewModels/ApplicationViewModel.cs

```diff
@@ -39,12 +38,27 @@ public ApplicationViewModel()
 
 		ShowCommand = ReactiveCommand.Create(() => ShowRequested?.Invoke(this, EventArgs.Empty));
 
-		TrayIcon = RuntimeInformation.IsOSPlatform(OSPlatform.OSX)
-			? new WindowIcon(AssetHelpers.GetBitmapAsset("avares://WalletWasabi.Fluent/Assets/WasabiLogo_white.ico"))
-			: new WindowIcon(AssetHelpers.GetBitmapAsset("avares://WalletWasabi.Fluent/Assets/WasabiLogo.ico"));
+		if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+		{
+			using var bitmap = AssetHelpers.GetBitmapAsset("avares://WalletWasabi.Fluent/Assets/WasabiLogo_white.ico");
+
+			if (bitmap is { })
+			{
+				TrayIcon = new WindowIcon(bitmap);
+			}
+		}
+		else
+		{
+			using var bitmap = AssetHelpers.GetBitmapAsset("avares://WalletWasabi.Fluent/Assets/WasabiLogo.ico");
+
+			if (bitmap is { })
+			{
+				TrayIcon = new WindowIcon(bitmap);
+			}
+		}
```

Comment:
Fix `AssetHelpers.GetBitmapAsset` to not allow null and this whole stuff can be reverted.

Reply:
done.